libpromises/cf3lex.l

```diff
@@ -77,6 +79,9 @@ line       ^.*$
 
 comment    #[^\n]*
 
+macro_if    ^@if\ version_after\([0-9]{1,10}\.[0-9]{1,10}\)
```

Comment:
I'd be OK with that change but it will be slightly harder to detect the boundaries mechanically.